rook/migration/v1.12/README.md

```diff
@@ -0,0 +1,224 @@
+# Upgrade rook-ceph to v1.12.x
+
+## Prerequisites
+
+- [ ] Check the state of rook-ceph so it is healthy before you begin:
+
+  ```bash
+  helm -n rook-ceph list --all
+  # should show all deployed
+  kubectl -n rook-ceph get jobs
+  # should show all completed
+  kubectl -n rook-ceph get pods
+  # should show all running or completed
+  kubectl -n rook-ceph get cephclusters
+  # should show PHASE = Ready and HEALTH = HEALTH_OK
+  ```
+
+## Prepare upgrade - _non-disruptive_
+
+_Nothing to prepare_
+
+## Apply upgrade - _non-disruptive_
```

Comment:
```suggestion
## Apply upgrade - _disruptive_
```
The apply upgrade section should _always_ be marked as disruptive, since it makes changes to a live environment.